src/pages/playlist/playlist.js

```diff
@@ -1,9 +1,51 @@
+import { Link } from "react-router-dom";
+import { usePlaylist } from "../../contexts/playlistContext/playlist-context";
+import { deleteSinglePlaylist } from "../../utilities/playlistsUtils";
 import "./playlist.css";
 
 const Playlist = () => {
+  const { playlistState, playlistDispatch } = usePlaylist();
+  const { playlists } = playlistState;
   return (
     <>
-      <p>This is playlist</p>
+      <main className="main-content">
+        <div className="playlist-heading">
+          <h1>My playlist</h1>
+          {/* <button>Create New Playlist</button> */}
+        </div>
+        <div className="all-playlist">
+          {playlists &&
```

Comment:
You can replace the additional check with (playlist?.map)

Reply:
Ok